distribution/pom.xml

```diff
@@ -73,6 +73,44 @@
       <version>0.5.1-SNAPSHOT</version>
       <type>pom</type>
     </dependency>
+    <dependency>
+      <groupId>org.junit.jupiter</groupId>
+      <artifactId>junit-jupiter</artifactId>
+      <version>5.9.0</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.codehaus.groovy</groupId>
+      <artifactId>groovy-all</artifactId>
+      <version>3.0.21</version>
+      <type>pom</type>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.sling</groupId>
+      <artifactId>org.apache.sling.testing.sling-mock-oak</artifactId>
+      <version>4.0.0-1.62.0</version>
+      <scope>test</scope>
+    </dependency>
```

Comment:
For the script to actually work on the JCR state prepared by test - we need a more "real" JCR_OAK resource resolver type